modules/account-lib/package.json

```diff
@@ -79,7 +79,7 @@
     "near-api-js": "^0.44.2",
     "noble-bls12-381": "0.7.2",
     "paillier-bigint": "3.3.0",
-    "protobufjs": "^6.8.9",
+    "protobufjs": "^6.11.3",
```

Comment:
why the changes to dependencies here? Don't see any changes to code in account-lib

Reply:
This change is already merged in master with other branches for a change. I'll remove this.